### PR TITLE
 implement a few size reduction changes

### PR DESCRIPTION
**Pull request by jerryneedell**
changes to reduce the code size.
disabled pylint check for "too-many"statements"  for the init() function.
without this I had to create a function to do some of the initializations. THis added 80 bytes to the code size.
The init() function is only a few statements over the 50 statement limit in pylint so it seemed better to override.

The savings for these changes is only about 140 bytes total but seems like it is worth doing.

I removed a few register initializations that were setting the registers to their default values.

Tested on RaspberryPi 4 with rfm69 bonnet and feather_m4_express with rfm69 featherwing.

### CODE REVIEW

File: adafruit_rfm69.py [hunk 1]
```diff
@@ -893,10 +894,9 @@ def receive(
                     # delay before sending Ack to give receiver a chance to get ready
                     if self.ack_delay is not None:
                         time.sleep(self.ack_delay)
-                    # send ACK packet to sender
-                    data = bytes("!", "UTF-8")
+                    # send ACK packet to sender (data is b'!')
                     self.send(
-                        data,
+                        b"!",
```
[the remark this answers is not in the document]
Comment (tannewt) [reply adafruit_rfm69.py:897]: Note that the original `bytes("!", "UTF-8") stores two strings plus the byte codes to convert them to a bytes object. `data = b"!"` will save those two strings and only cost the assign byte code assuming the name `data` is used elsewhere.

**Comment by brentru** [comment]
Ok - holding off until that issue is cleared.

**Comment by jerryneedell** [comment]
Still investigating the SPI baud rate issue, but wanted to get some of these updated in place. Refactored some code to eliminate a function that was used for writing to the FIFO. now done via the general write_from().

**Comment by jerryneedell** [comment]
@brentru after the discussion at yesterdays CP weekly, I think this is ready to go. There are some long term changes that may be worth looking into but I don't think we have to wait. Let me know if you have any questions about this or if you want any additional changes.

**Review by brentru — APPROVED** [review]
LGTM!

**Pull request merged**